### PR TITLE
Bpf120 dolt insurance pitch

### PR DESCRIPTION
**Pull request by bpf120**
starting to riff

### CODE REVIEW

File: 01-24-2023-dolt-insurance-pitch.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+---
+
+
+
+
+Email #1
+Hello,
+
+Are you concerned about the potential damage that can be caused by an administrator's accidental deletion or alteration of data in your MySQL database? Dolt replication can provide peace of mind by creating a replica of your database that can be used for recovery in the event of a data loss or corruption.
```
Comment (timsehn) [inline 01-24-2023-dolt-insurance-pitch.md]: This needs to be tightened up a bit. Fewer words.

**Review by timsehn — COMMENTED** [review]
Iterate a bit on that opener. It's a lot of words.

File: 01-24-2023-dolt-insurance-pitch.md (hunk 1 above)
Comment (bpf120) [reply 01-24-2023-dolt-insurance-pitch.md]: done

File: 01-24-2023-dolt-insurance-pitch.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+---
+
+
+
+
+Email #1
+Hello,
+
+Everyone has horror stories about a database being broken with a bad query. A single mistake disrupts the functionality of your database and systems for maybe months.
```
Comment (timsehn) [inline 01-24-2023-dolt-insurance-pitch.md]:
```suggestion
Everyone has horror stories about a database broken by a bad query. A single mistake breaks your application for hours, days, or even [months](https://www.dolthub.com/blog/2022-04-14-atlassian-outage-prevention/).  
```

File: 01-24-2023-dolt-insurance-pitch.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+
+
+
+Email #1
+Hello,
+
+Everyone has horror stories about a database being broken with a bad query. A single mistake disrupts the functionality of your database and systems for maybe months.  
+
+Dolt solves this problem.
```
Comment (timsehn) [inline 01-24-2023-dolt-insurance-pitch.md]:
```suggestion
Dolt solves this problem with a special type of replica you add to an existing MySQL deployment. No migration required.
```

Do we need a little more info here?

**Review by timsehn — COMMENTED** [review]
Some thoughts

File: 01-24-2023-dolt-insurance-pitch.md (hunk 2 above)
Comment (bpf120) [reply 01-24-2023-dolt-insurance-pitch.md]: links arent great in cold emails but we can reference this

**Pull request merged**